packages/console/src/pages/Applications/index.tsx

```diff
@@ -49,6 +53,19 @@ const Applications = () => {
   const isLoading = !data && !error;
   const [applications, totalCount] = data ?? [];
 
+  const appToShowInGuide = useMemo(() => {
+    if (isShowingGuide && appId) {
+      return applications?.find(({ id }) => id === appId);
+    }
+  }, [applications, isShowingGuide, appId]);
```

Comment:
This code appears to be risky. Because the user may not create an application on the first page, but the `applications` data only includes data from the current page.

Reply:
Thanks. Code updated and now app guide is placed under app details page, which makes more sense